clients/cobol-lsp-vscode-extension/src/test/suite/lsp.spec.test.ts

```diff
@@ -45,8 +45,9 @@ suite('Integration Test Suite', () => {
     }).timeout(10000).slow(4000);
 
     test('TC152049 Navigate through definitions', async () => {
-        await helper.sleep(500);
+        await helper.sleep(1000);
         const result: vscode.Location[] = await vscode.commands.executeCommand('vscode.executeDefinitionProvider', editor.document.uri, new vscode.Position(28, 24));
+        await helper.sleep(2000);
```

Comment:
do we really need sleep after synchronous command?

Reply:
well ! I have no clue why these tests fail without sleep. I agree it would need some investegation.

Reply:
Updated!